dpm/api/env.py

```diff
@@ -97,3 +98,14 @@ def get_secret(self, key: str) -> str:
                 "You must invoke initialize with the appropriate parameters"
             )
         return self.secrets_client.get_secret(key)
+
+    def to_env(self):
+        import json
+
+        properties = self.dpm_client.get_dynamic_properties()
+        for k, v in properties.items():
+            if isinstance(v, dict):
+                for _, v in v.items():
+                    os.environ[k] = v
```

Comment:
Given the "rule" that ENV VARS should always have priority when defined, I'd add:
`if not k in os.environ:
  os.environ[k] = v`
here and line 111

Reply:
Good catch, I will update for discussion with others.

Reply:
Why are you doing this? Pushing all properties to the os??!! No please.

Reply:
Are you trying to emulate IOC? This is really odd.